api/api.yaml

```diff
@@ -39,6 +39,13 @@ paths:
           collectionFormat: pipes
           items:
             type: string
+        - name: extraFacets
```

Comment:
Working on this

Reply:
Don't we need to add this to the `/exportUrl` request as well so that we create the proper query?

Reply:
Yeah, I am doing that in a separate PR

Reply:
Cool sorry, when you do that do you think you could merge make a model definition wrapping those two fields so that we can share it between the two endpoints? I'm starting to think we should just make `/facets` a POST. Thoughts @melissachang?